README.md

```diff
@@ -123,3 +123,20 @@ two json files have been loaded, and the related checkbox will enable/disable th
 
 The functionalities for the graph drawing and its behavior are developed in javascript and are avaialble
 at the following [repository](https://github.com/oda-hub/renku-aqs-graph-library/).
+
+
+# Installation of the plugin
+
+Currently, the plugin is developed using the version `2.1.0` of [renku-python](https://github.com/SwissDataScienceCenter/renku-python). Please make sure such version is installed by running:
```

Comment:
it's good to have the note of course, but could you also make a check for renku version while importing the plugin? With an instructive error message

Reply:
how could I access to the renku version specified within `setup.py`? So to compare it with the installed one...

Reply:
Usually something like this works:

```python
from packaging import version
import renku

version.parse(renku.__version__) > version.parse("2.1.0")

```

Reply:
Or you mean version in own dependency? 
Well, in principle, if installed nominally, it should never happen that the version of the dependency deviates. I.e. there is no need for extra check or readme.

But if the problem justifies addition in the README, you might as well add an extra check in the code.

Reply:
> Or you mean version in own dependency? 

Yes, that is what I meant. To check if the installed version is different from the one listed in the `requirements`/`install_requires`

Reply:
```
from pip._vendor import pkg_resources


_package_name = 'somepackage'
_package = pkg_resources.working_set.by_key[_package_name]

print([str(r) for r in _package.requires()])  # retrieve deps from setup.py
```

Reply:
Maybe add in readme that the dependency is in principle guaranteed with normal installation, and version is checked in runtime.

Reply:
Within the `__init__`, I now compare the two renku versions. So the check will be performed when importing the plugin